src/features/search/pages/modals/DatesHoursModal/DatesHoursModal.tsx

```diff
@@ -42,7 +40,7 @@ type DatesHoursModalFormData = {
   selectedDateChoice: DATE_FILTER_OPTIONS
   selectedDate?: Date
   hasSelectedHours: boolean
-  selectedHours?: Range<number>
+  selectedHours: Range<number>
```

Comment:
pourquoi ce n'est pas des `Hour` ?

```suggestion
  selectedHours: Range<Hour>
```